calebwagner/uteach-ilearn-server
# Cjw profile setting

#### Changes Made
1. Modified file `models/meetings.py` to include `MM/DD/YEAR` value instead of timestamp.
1. Modified file `fixutres/meetings.json` to include updated model `MM/DD/YEAR` attribute.
   ​
#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout cjw-profile-settings
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > Confirm file modifications are present as indicated above.
